crates/sui-types/src/unit_tests/base_types_tests.rs

```diff
@@ -353,12 +362,16 @@ fn test_move_package_size_for_gas_metering() {
     let serialized = bcs::to_bytes(&package).unwrap();
     // If the following assertion breaks, it's likely you have changed MovePackage's fields.
     // Make sure to adjust `object_size_for_gas_metering()` to include those changes.
-    assert_eq!(size + 2, serialized.len());
+    assert_eq!(size - 2, serialized.len());
```

Comment:
is this the correct thing to do to change the test? from what i see, the object_size_for_gas_metering seems to be an approx to begin with, and no need to tweak there. 

```
        let meta_data_size = size_of::<Owner>() + size_of::<TransactionDigest>() + size_of::<u64>();
```

this should take into account the size of owner aka the increased address size